pkgs/development/python-modules/djangorestframework-stubs/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, django-stubs
+, fetchPypi
+, mypy
+, pythonOlder
+, requests
+, types-pyyaml
+, types-requests
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "djangorestframework-stubs";
+  version = "3.14.2";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-SpFpJT0tRohaAHwIXWTpbvyKeL2TjHR+mtBlRyXE59A=";
+  };
+
+  propagatedBuildInputs = [
+    django-stubs
+    mypy
+    requests
+    types-pyyaml
+    types-requests
+    typing-extensions
+  ];
+
+  doCheck = false;
```

Comment:
Why do you disable tests?

Reply:
Thanks, yes that's not needed. Fixed!

Reply:
We might be able to run the test by fetching the source from github.
However, since this seems to be a stub, it would be better to revert and leave a comment like "it's a stub, so we don't run the test".

Reply:
Thanks, I've added the fetch from GitHub and added the test deps.